doc/suda.txt

```diff
@@ -40,10 +40,21 @@ Make sure that the following shows 1.
 =============================================================================
 USAGE						*suda-usage*
 
-Add "suda://" prefix (see |g:suda#prefix|) to the filename and use Vim's
-builtin |read|, |edit|, |write|, |saveas| commands.
+Use |SudaRead| and |SudaWrite| commands like:
+>
+	" Open a current file with sudo
+	:SudaRead
```

Comment:
💭 [IMO]
User may confuse: "I open a current file with sodo via `:SudaRead`, then edit, and I should use `:SudaWrite` to save it."

```
This reopens a current file with `suda://` prefix.  So you can save it just `:w`.
```

Reply:
What do you think the best interface here?

Reply:
Interface is OK.
I fear that users will make the mistake looking at the following sequence.

```
" Open a current file with sudo
:SudaRead

" Save a current file with sudo
:SudaWrite
```

I thought it would be good if there was a note.